styles.css

```diff
@@ -0,0 +1,63 @@
+h1 {
```

Comment:
@filippia Just a heads up... you could have combined these first two CSS statements into one like so:
```css
h1, h2 {
  text-align: center;
}
```